doc/lua_api.md

```diff
@@ -564,17 +564,22 @@ Example:
 
     default_cobble.png^[crack:10:1
 
-#### `[combine:<w>x<h>:<x1>,<y1>=<file1>:<x2>,<y2>=<file2>:...`
-
-* `<w>`: width
-* `<h>`: height
-* `<x>`: x position, negative numbers allowed
-* `<y>`: y position, negative numbers allowed
-* `<file>`: texture to combine
+#### `[combine:<w>x<h>:<x1>,<y1>,<w1>=<file1>:<x2>,<y2>,<w2>=<file2>:...`
 
 Creates a texture of size `<w>` times `<h>` and blits the listed files to their
 specified coordinates.
 
+Note: the output texture size may vary if one or more of `<wN>` is provided.
+The aspect ratio is preserved.
+
+* `<xN>`: X offset of insertion, negative numbers allowed
+* `<yN>`: Y offset of insertion, negative numbers allowed
+* `<wN>` (optional): Expected texture width
+   * Scales the output image if `<wN> != input texture width` to perform lossless blit.
```

Comment:
scales it how? there should also be an example

Reply:
I suppose `copyToScaling` uses linear scaling.
If `<wN>` is 16, but a 32x32 texture is provided, the output will be scaled to 2x the originally specified size.
^ should I perhaps add this sentence for clarification?

Reply:
I meant how exactly it will affect the output texture size, so that'd be good to clarify.

Reply:
Added a more detailed explanation. Please let me know whether this good enough.